### ISSUE TITLE
Won't install under Hassioos 1.5

### ISSUE THREAD

**Issue by faspina** [danielwelch/hassio-dropbox-sync#6]
This addon was working great , until I upgraded to the new hassio.os 1.5 , now I cannot get it installed from the repository.

Here is the system log under the hassio menu
18-07-20 14:38:36 INFO (SyncWorker_10) [hassio.docker.interface] Pull image dwelch2101/dropbox-sync-aarch64 tag 1.2.0.
18-07-20 14:38:38 ERROR (SyncWorker_10) [hassio.docker.interface] Can’t install dwelch2101/dropbox-sync-aarch64:1.2.0 -> 500 Server Error: Internal Server Error (“readlink /var/lib/docker/overlay2: invalid argument”).

**Comment by faspina** [comment]
Same issue on HassOS 1.7

**Comment by danielwelch** [comment]
Image may just need to be rebuilt. I’ll look into it and see if I need to change the base image or anything.

**Comment by danielwelch** [comment]
Is this still an issue? I’ve heard others who have upgraded to HassOS without issue. If still an issue, can you tell me more details about your setup? We’ve run into problems with NUC before.

**Comment by faspina** [comment]
Yes it seems to still be happening. I have HASSOS host 1.9 and HA 75.1 . 
Here is the error.


18-08-05 20:16:50 INFO (SyncWorker_2) [hassio.docker.interface] Pull image dwelch2101/dropbox-sync-aarch64 tag 1.2.0.
18-08-05 20:16:52 ERROR (SyncWorker_2) [hassio.docker.interface] Can't install dwelch2101/dropbox-sync-aarch64:1.2.0 -> 500 Server Error: Internal Server Error ("readlink /var/lib/docker/overlay2: invalid argument").

**Comment by faspina** [comment]
I probably should note it was working before I went with HASSOS a few 
weeks ago.

**Comment by danielwelch** [comment]
Thanks for clarifying. Still interested what hardware your using as others on the forums have installed successfully on HassOS.

**Comment by danielwelch** [comment]
https://github.com/home-assistant/hassio/issues/582 seems related

**Comment by faspina** [comment]
Yes I did restore from backup and the addon would not work so I deleted it then try to add it back. I can do what you suggest but it will be a while. I figured something was corrupt , is there now way to cleanout the overlay data in hassos

**Comment by danielwelch** [comment]
I don’t know of any way to do that through the UI. You could muck around on the command line by attaching to docker containers, but I haven’t looked into the file structures or what you would need to do.

**Comment by faspina** [comment]
i am copying backup to dropbox manually now  so its not that big of a deal. If I have time this weekend I will redo my hassos install see if I can do a cleaner restore
